internal/exec/engine.go

```diff
@@ -53,6 +53,7 @@ type Engine struct {
 	FetchTimeout      time.Duration
 	Logger            *log.Logger
 	Root              string
+	OEM               string
```

Comment:
You're adding OEM to the Engine struct without updating `internal/main.go` to pass it in.